lib/patches/before/openssl.rb

```diff
@@ -0,0 +1 @@
+require "openssl-stubs" unless ENV['TRUFFLERUBY_OPENSSL_ENABLED']
```

Comment:
I don't understand why we need `TRUFFLERUBY_OPENSSL_ENABLED`? If someone requires `openssl` then it's fine to let it fail as it is currently if Sulong isn't there.